packages/desktop/views/dashboard/wallet/views/WalletView.svelte

```diff
@@ -1,8 +1,16 @@
 <script lang="ts">
     import { AssetList, Overflow, Pane, ReceiveAddressButton } from '@ui'
     import { AccountSummary, AccountActivity, SendButton } from '@components'
-    import { selectedWalletAssets } from '@core/wallet'
+    import { mainAccountId, selectedWalletId, selectedWalletAssets } from '@core/wallet'
     import features from '@features/features'
+    import { onMount } from 'svelte'
+    import { updateActiveWalletPersistedData } from '@core/profile'
+
+    onMount(() => {
```

Comment:
I think this is not correct. I would have to update it 3 times
- When the account is created for the first time (handleEventNewOutput)
- When the accounts exist and there is no value (buildWalletState)
- When it can be changed in account management (account management)